BetaUtils.Core/Exceptions/Middleware/TrackedExceptions.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Http;
+
+namespace BetaUtils.Core.Exceptions.Middleware;
+
+public static class TrackedExceptions
+{
+    private static Dictionary<Type, int> _exceptions = DefaultExceptions();
+
+    /// <summary>
+    /// Add exception with status code to tracked exceptions. Only one exception of same type may exist in dictionary.
+    /// </summary>
+    /// <returns>True if exception added successfully.</returns>
+    public static bool Add<TException>(int statusCode) where TException : Exception
+    {
+        if (_exceptions.ContainsKey(typeof(TException)))
```

Comment:
It is not necessary, in ticket was no stated whether middleware should be usable in code implementation or not so I've created it as "just in case". Should have checked for more details about issue. 😁

Reply:
Also if its going to be used on startup, non of the methods are needed except `default` method that will be containing list of exceptions that middleware is handling. Am i missing something?